Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -84,16 +100,45 @@ define([
         VertexArray,
         BlendingState,
         DepthFunction,
+        ImageryState,
         PerInstanceColorAppearance,
         Primitive,
+        TileBoundingRegion,
+        TileSelectionResult,
         ClippingPlaneCollection,
         GlobeSurfaceTile,
         ImageryLayer,
         QuadtreeTileLoadState,
         SceneMode,
-        ShadowMode) {
+        ShadowMode,
+        TerrainFillMesh) {
     'use strict';
 
+    /**
+     * The strategy to use to fill the space of tiles that are selected for rendering but that
+     * are not yet loaded / renderable.
+     * @private
+     */
+    var MissingTileStrategy = {
```

Comment:
Will these options just be replaced with the one right/passable strategy?

Reply:
Yeah I'll probably get rid of this. There are some other tweakables I'll leave in, though.